SCRIPTING

```diff
@@ -39,7 +39,9 @@ Overview
     global setup    -- called during thread setup
     global init     -- called when the thread is starting
     global request  -- called to generate the HTTP request
-    global response -- called with HTTP response data
+    global response -- called with HTTP response header and body
+                              (with --lua-dont-pass-body command line option
+                               the body is omitted)
```

Comment:
I find this phrasing a bit confusing. How about "(only the response header is passed if --lua-dont-pass-body is enabled)? Or maybe "(the body is omitted if --lua-dont-pass-body is enabled)"?

Reply:
I'll rephrase to "(body does not contain actual response data if the command line option --lua-dont-pass-body is used)".